bryft/src/pages/HomePage/HomePage.js

```diff
@@ -0,0 +1,35 @@
+import "./HomePage.css";
+import React from 'react';
+import Navbar from "../../components/Navbar/Navbar";
+import backdrop from './royce-backdrop.png';
+import icon1 from './icon1.png';
+import icon2 from './icon2.png';
+import icon3 from './icon3.png';
+import { Link } from "react-router-dom";
+
+function HomePage() {
+    return (
+        <React.Fragment>
+            <Navbar />
+            <img className="backdrop" src={backdrop} alt="Backdrop" />
+            <h1>BETTER TRANSPORT FOR BRUINS.</h1>
+            <Link to="/rides"><div className="textbox t1">Create a group</div></Link>
```

Comment:
For lines 16 and 17, the words go out of the box when a user zooms in or zooms out. Look at this: 
<img width="1399" alt="image" src="https://github.com/Alankvuong/Bryft/assets/63077857/f039aac0-30e9-44c9-ac65-fa8279d14f58">
<img width="1399" alt="image" src="https://github.com/Alankvuong/Bryft/assets/63077857/28af0d25-7b91-4382-80a7-561b9e6a11ee">

Can we set a standard width for the box and a max-width so that the words will stay center aligned regardless of the window size?